itou/templates/companies/card.html

```diff
@@ -37,8 +37,22 @@ <h2 class="visually-hidden">Actions rapides</h2>
                                     </a>
                                 </div>
                             {% endif %}
-                            <div class="form-group col-12 col-lg d-lg-flex justify-content-lg-end">
-                                {% include 'includes/copy_to_clipboard.html' with content=siae_card_absolute_url text="Copier le lien de cette fiche" placement="bottom" css_classes="btn btn-lg btn-ico btn-link-white btn-block w-lg-auto" %}
+                            <div class="form-group col-12 col-lg d-flex justify-content-center justify-content-lg-end">
+                                <button class="btn btn-lg btn-ico btn-link-white" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Ouverture et fermeture du menu des actions complémentaires">
+                                    <i class="ri-more-2-line ri-lg font-weight-normal" aria-hidden="true"></i>
+                                    <span class="d-lg-none">Autres actions</span>
+                                </button>
+                                <ul class="dropdown-menu">
+                                    <li>
+                                        {% include 'includes/copy_to_clipboard.html' with content=siae_card_absolute_url text="Copier le lien de cette fiche" css_classes="dropdown-item btn-ico btn-link" %}
+                                    </li>
+                                    <li>
+                                        <a href="https://tally.so/r/m62GYo?jobdescriptionid={{ siae.pk }}&amp;userid={{ user.id }}" class="dropdown-item btn-ico btn-link" target="_blank">
```

Comment:
```suggestion
                                        <a href="https://tally.so/r/m62GYo?jobdescriptionid={{ siae.pk }}&amp;userid={{ request.user.id }}" class="dropdown-item btn-ico btn-link" target="_blank">
```
pour éviter d'avoir à toucher le contexte de la vue.
Par contre, tu peux tout à fait accéder à cette page sans être authentifié: https://emplois.inclusion.beta.gouv.fr/company/6893/card et tu auras donc `(request.)user.id` qui sera vide: à voir ce qu'on souhaite avoir dans ce cas ?

Reply:
On souhaite ne pas envoyer le paramètre. Je reprends le sujet, merci pour la revue :)